src/components/modals/WalletListModal.js

```diff
@@ -76,11 +80,23 @@ export function WalletListModal(props: Props) {
     excludeCurrencyCodes
   } = props
 
+  // #region Constants
+
   const account = useSelector(state => state.core.account)
   const [searching, setSearching] = useState(false)
   const [searchText, setSearchText] = useState('')
+  const [linkedBankAccounts, setLinkedBankAccounts] = useState()
+  useAsyncEffect(async () => {
+    const wyreClient = await makeWyreClient({ account })
+    if (wyreClient.isAccountSetup) {
+      setLinkedBankAccounts(await wyreClient.getCryptoPaymentMethods())
+    }
+  })
```

Comment:
Missing deps array. `useEffect` has a lint-rule which warns you when you do thi, but `useAsyncEffect` does not. I wonder if there is a way to add our rule for our custom `useAsyncEffect`.